app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,21 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+<div class="<%= dom_id(sender) %>_<%= dom_id(recipient) %>_follow_unfollow">
```

Comment:
I love that you added a div to capture the unique ids

Reply:
great job utilizing a concise and efficient solution to this problem!